json_dynamic_widget/lib/src/components/json_widget_registry.dart

```diff
@@ -80,7 +81,7 @@ class JsonWidgetRegistry {
   final JsonWidgetRegistry? _parent;
   final _values = <String?, dynamic>{};
   late List<ArgProcessor> _argProcessors;
-
+  final Widget Function(Object error, BuildContext context)? onBuildWidgetFailed;
```

Comment:
Woth to pass also the Stacktrace and let's type error as `dynamic` just to be consistent

Reply:
i can try it

Reply:
pls check it out the update!